internal/runtime/init/init.go

```diff
@@ -130,6 +130,13 @@ func LoadPaths(paths []string,
 	var result LoadPathsResult
 	var err error
 
+	var targzs, nonTargzs []string
+	if !asBundle {
+		targzs, nonTargzs = split(paths)
+		paths = targzs
+		asBundle = true
+	}
+
 	if asBundle {
```

Comment:
This will never be false with the code above.

Reply:
As I explained above asBundle will not be set true in all cases. only in certain cases.

Reply:
No, it will always be true when this code is reached.